tests/test_mirror_ipv6_combined.py

```diff
@@ -142,6 +142,78 @@ def remove_mirror_acl_rule(self, table, rule):
         tbl._del(table + "|" + rule)
         time.sleep(1)
 
+    def test_CombinedMirrorTableCreation(self, dvs, testlog):
+        self.setup_db(dvs)
+
+        acl_table = "MIRROR_TABLE"
+        ports = ["Ethernet0", "Ethernet4"]
+
+        # Create the table
+        self.create_acl_table(acl_table, ports, "MIRROR")
+
+        # Check that the table has been created
+        tbl = swsscommon.Table(self.adb, "ASIC_STATE:SAI_OBJECT_TYPE_ACL_TABLE")
+        table_entries = [k for k in tbl.getKeys() if k not in dvs.asicdb.default_acl_tables]
+        assert len(table_entries) == 1
+
+        # Get the data from the table
+        table_id = table_entries[0]
+        status, attributes = tbl.get(table_id)
+        assert status
+
+        # TODO: Refactor mirror table tests so that these attributes can be shared between tests for v4, v6, and
+        # dscp mirror tables.
+        expected_sai_attributes = [
```

Comment:
I feel that 'expected' is a bit misleading to encourage people to think all attributes should be there? Or is it just my English? :-)

Reply:
Thanks for catching this, I've added an extra check to make sure they're all there! :)